examples/nextjs/src/components/my-page.js

```diff
@@ -18,53 +18,55 @@ import { withExperiments } from "@dotcms/experiments";
  * you can ignore or remove the experiment-related code and imports.
  */
 const experimentConfig = {
-  apiKey: process.env.NEXT_PUBLIC_EXPERIMENTS_API_KEY, // API key for experiments, should be securely stored
-  server: process.env.NEXT_PUBLIC_DOTCMS_HOST, // DotCMS server endpoint
-  debug: process.env.NEXT_PUBLIC_EXPERIMENTS_DEBUG // Debug mode for additional logging
+    apiKey: process.env.NEXT_PUBLIC_EXPERIMENTS_API_KEY, // API key for experiments, should be securely stored
+    server: process.env.NEXT_PUBLIC_DOTCMS_HOST, // DotCMS server endpoint
+    debug: process.env.NEXT_PUBLIC_EXPERIMENTS_DEBUG, // Debug mode for additional logging
 };
 
 // Mapping of components to DotCMS content types
 const componentsMap = {
-  webPageContent: WebPageContent,
-  Banner: Banner,
-  Activity: Activity,
-  Product: Product,
-  Image: ImageComponent
+    webPageContent: WebPageContent,
+    Banner: Banner,
+    Activity: Activity,
+    Product: Product,
+    Image: ImageComponent,
+    Empty: () => <div class="relative w-full bg-gray-200 h-12 flex justify-center items-center overflow-hidden">Dont have a component for this contentlet.</div>,
```

Comment:
Keep this consistent with the external file.

Reply:
Well... maybe is cool to show that we can do inline stuff.

But also is important to show the content type name here.

Reply:
Changed!